README.md

```diff
@@ -38,7 +36,7 @@ defmodule MyApp do
   def start(_type, _args) do
     # List all child processes to be supervised
     children = [
-      Plug.Cowboy.Adapters.child_spec(scheme: :http, plug: MyRouter, options: [port: 4001])
+      Plug.Cowboy.child_spec(scheme: :http, plug: MyRouter, options: [port: 4001])
```

Comment:
1. how about `{Plug.Cowboy, scheme: :http, plug: MyRouter, options: [port: 4001]}`?
2. we require Elixir v1.4 but I don't think there's a built-in way to run it pre-child specs, is there?

Reply:
I think we can probably depend on Elixir 1.5 here, the 1.4 support here is only because that's what was in the mix.exs file I copied over from plug!